cluster/gce/util.sh

```diff
@@ -3464,13 +3469,6 @@ function check-resources() {
     return 1
   fi
 
-  if [[ ${GCE_PRIVATE_CLUSTER:-} == "true" ]]; then
```

Comment:
Why did you remove this one?

Reply:
It was used to detect whether we didn't leak cloud nat.

Reply:
check-resources is used only here:

https://github.com/kubernetes/kubernetes/blob/master/cluster/gce/util.sh#L2323

It checks if there is no other cluster running in the network with the same name and asks if kube-down should be called.

It's a bug that we check there for existence of cloud-nat, which is created per network and not per cluster.

If we will leave this here, any attempt to create a second private cluster in the same network (e.g. in kubemark) will ask for deleting the previous one which is incorrect.